paying_for_college/models.py

```diff
@@ -400,8 +400,10 @@ def total_fields(field_list):
     def cost_error(self):
         """Return 1 or 0: Is total-cost less than tuition?"""
         url_data = self.parsed_url
-        if url_data and url_data['totl'] != '' and url_data['tuit'] != '':
-            if int(url_data['totl']) < int(url_data['tuit']):
+        if url_data:
+            totl = url_data.get('totl') or 0
+            tuit = url_data.get('tuit') or 0
```

Comment:
Oh yeah, good call on assigning them a value, since converting `None` to `int` would've inevitably led to an error. Sorry I missed that in my review. 

Not to be too nitpicky, but I think it's more canonical to do `url_data.get('totl', 0)` to assign a default value.